core/src/main/res/values/strings.xml

```diff
@@ -365,7 +367,9 @@
     <string name="pref_favorite_keeps_episodes_sum">Keep episodes when they are marked Favorite</string>
     <string name="pref_favorite_keeps_episodes_title">Keep Favorite Episodes</string>
     <string name="playback_pref">Playback</string>
+    <string name="playback_pref_sum">Headphone, bluetooth, playback control, skipping, queue...</string>
```

Comment:
I don't think users get an idea of what the settings do when writing `Headphone`, `bluetooth`. Maybe just Interruptions?

Reply:
`Headphone controls, Skip intervals, Queue & playlist`?